paket.dependencies

```diff
@@ -1,5 +1,6 @@
 source https://nuget.org/api/v2
 
+nuget Nuget.CommandLine
```

Comment:
Only to actually push the package (or can you use paket for that?)  "build PublishPackage" didn't seem to actually push it to nuget but maybe it was missing the api key